src/condition.cpp

```diff
@@ -729,6 +729,26 @@ void conditional_t<T>::set_npc_allies( const JsonObject &jo )
     };
 }
 
+
+template<class T>
+void conditional_t<T>::set_npc_allies_global( const JsonObject &jo )
+{
+    int_or_var iov = get_int_or_var( jo, "npc_allies_global" );
+
+    condition = [iov]( const T & d ) {
+        const auto all_npcs = overmap_buffer.get_overmap_npcs();
+        std::vector<shared_ptr_fast<npc>> filtered_npcs{};
```

Comment:
Is this used anywhere?

Reply:
Whoops, no it's not. That was from an earlier thing I was trying.